scripts/generate-api-docs.js

```diff
@@ -63,7 +63,7 @@ const format_type = (type) => {
                 rst[key] = format_type(shape[key])
             })
 
-            return JSON.stringify(rst, null, 2).replaceAll('\n', '<br/>')
+            return JSON.stringify(rst, null, 2).replace(/\n/g, '<br/>')
```

Comment:
I looked up why you might have switched from `.replaceAll` to `.replace`, which led me [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replaceAll#non-global_regex_throws). It seems odd that you always have to pass a global flag wo whatever regex you pass to `.replaceAll`, since a non-global replace-all action seems to be an impossible combination to me.